README.md

```diff
@@ -1,24 +1,143 @@
-# README
+# 📗 Table of Contents
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)  
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-Things you may want to cover:
+# 📖 Take my money <a name="about-project"></a>
 
-* Ruby version
+> This application allows you to register and log in to manage your finances, you can invest your money in the different categories available.
+## 🛠 Built With <a name="built-with"></a>
 
-* System dependencies
+### Tech Stack <a name="tech-stack"></a>
 
-* Configuration
+> This project is made with Ruby on Rails.
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/command_line.html">Rails Server</a></li>
+  </ul>
+</details>
 
-* Database creation
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* Database initialization
+### Key Features <a name="key-features"></a>
 
-* How to run the test suite
+- **Users can create categories**
+- **Users can create transactions**
+- **Users can register**
 
-* Services (job queues, cache servers, search engines, etc.)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-* Deployment instructions
+## 🚀 Live Demo <a name="live-demo"></a>
 
-* ...
+> Add a link to your deployed project.
+- [Live Demo Link](https://take-my-money-app.onrender.com/)
+
```

Comment:
- [ ] The same problem still exists, Please provide a valid and functioning deployment link.

![image](https://user-images.githubusercontent.com/68796232/219902086-7cdc3a77-a346-430b-9abb-afebc1dadff5.png)